pkg/virt-launcher/virtwrap/converter/arch/amd64.go

```diff
@@ -130,3 +122,13 @@ func (converterAMD64) ConvertWatchdog(source *v1.Watchdog, watchdog *api.Watchdo
 func (converterAMD64) SupportPCIHole64Disabling() bool {
 	return true
 }
+
+func (converterAMD64) GetVideoType(vmi *v1.VirtualMachineInstance, isEFI bool) string {
+	if vmi.Spec.Domain.Devices.Video != nil {
+		return vmi.Spec.Domain.Devices.Video.Type
+	}
+	if isEFI {
+		return "bochs"
+	}
+	return "vga"
+}
```

Comment:
this function is almost identical across archs ..

Reply:
ah, please don't drop. I was just thinking if we can generalize this function and each will set it own defaults, but seems like an overhead right now